pkg/apiserver/apiserver.go

```diff
@@ -27,8 +27,11 @@ import (
 	"path"
 	rt "runtime"
 	"strconv"
+	"strings"
 	"time"
 
+	"k8s.io/client-go/pkg/util/sets"
+	"k8s.io/client-go/pkg/version"
```

Comment:
Eh..I'm using goimports locally and I guess it imports based on alphabetical order.